<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tartm</class>
 <widget class="QWidget" name="tartm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>741</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>741</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TAR TileMap Creation</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>176</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QLabel" name="comprLabel">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tarRadio">
      <property name="text">
       <string>Uncompressed (.tar)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="targzRadio">
      <property name="text">
       <string>GZip (.tar.gz)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tarbz2Radio">
      <property name="text">
       <string>BZip2 (.tar.bz2)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tarxzRadio">
      <property name="text">
       <string>LZMA (.tar.xz)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="tileView"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="saveBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>9</y>
     <width>541</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QListWidget" name="resourceView">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
